homeassistant/components/image_processing/openalpr_cloud.py

```diff
@@ -109,12 +109,13 @@ def async_process_image(self, image):
         websession = async_get_clientsession(self.hass)
         params = self._params.copy()
 
-        params['image_bytes'] = str(b64encode(image), 'utf-8')
+        image_to_send = {}
+        image_to_send['image_bytes'] = str(b64encode(image), 'utf-8')
```

Comment:
```
body = {
  'image_bytes': ...
}
```

Reply:
Thank you for the review. I've made the requested changes.